proposals/NNNN-add-preferred-FPS.md

```diff
@@ -0,0 +1,128 @@
+# Add preferred FPS to VideoStreamingCapability
+
+* Proposal: [SDL-NNNN](NNNN-add-preferred-FPS.md)
+* Author: [Shinichi Watanabe](https://github.com/shiniwat)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / RPC]
+
+## Introduction
+
+This proposal extends SDL video streaming feature by adding preferred FPS to VideoStreamingCapability, so that frame rate of video streaming can be adjusted per OEM.

```

Comment:
```suggestion
This proposal extends SDL video streaming feature by adding preferred FPS to `VideoStreamingCapability`, so that frame rate of video streaming can be adjusted per OEM.
```